_includes/v23.1/sql/shell-shortcuts.md

```diff
@@ -1,30 +1,37 @@
 Note: macOS users may need to manually enable Alt-based shortcuts in their terminal configuration. See the section [macOS terminal configuration](#macos-terminal-configuration) below for details.
 
-| Shortcut                | Description                                                                        |
+| Shortcut                | Description |
 |-------------------------|------------------------------------------------------------------------------------|
-| Tab                     | Use [context-sensitive command completion](#tab-completion).                                          |
-| Ctrl+C                  | Clear / Cancel the input.                                                          |
-| Ctrl+M, Enter           | New line / enter.                                                                  |
-| Ctrl+F, Right arrow     | Forward one character.                                                             |
-| Ctrl+B, Left arrow      | Backward one character.                                                            |
-| Alt+F, Ctrl+Right arrow | Forward one word.                                                                  |
-| Alt+B, Ctrl+Left arrow  | Backward one word.                                                                 |
-| Ctrl+L                  | Refresh the display.                                                               |
-| Delete                  | Delete the next character.                                                         |
-| Ctrl+H, Backspace       | Delete the previous character.                                                     |
+| Tab                     | Use [context-sensitive command completion](#tab-completion). |
+| Ctrl+C                  | Clear / Cancel the input. |
```

Comment:
```suggestion
| Ctrl+C                  | Clear/cancel the input. |
```
Noticed a few instances of spaces before and after a "/" and a few with no spaces. My instinct is to go no space here and for the one below.